### PR TITLE
Nix executor [PLEN-200 follow up]

### PR DESCRIPTION
**Pull request by adamma-da**

### CODE REVIEW

**Review by lucb-da — APPROVED** [review]

File: shell.nix
Comment (lucb-da) [inline shell.nix:1]: I don't think a `shell.nix` is required.
It is only useful if we expect users to start a `nix-shell` to work on the repository.

**Review by lucb-da — CHANGES_REQUESTED** [review]
Not sure if the nix config is as minimal as it can be.

**Comment by adamma-da** [comment]
@lucb-da Changes proposed by you have been implemented :)

File: .circleci/config.yml [hunk 1]
```diff
@@ -42,7 +45,7 @@ jobs:
            set -euo pipefail
            echo "Success!"
   release-chart:
-    executor: chart-releaser
+    executor: nix
     steps:
       - build-cr
```
Comment (lucb-da) [inline .circleci/config.yml:50]: Is the `build-cr` step still need, or even working ?
It installs the dependencies, but it should not be required anymore if they are provided by nix.
And I am not sure `apk` is available in the nix environment.

**Pull request closed without merging**